### DISCUSSION TITLE
Bentley MHI2_ER_BYG24_P5160_MU1445_PATCH

### DISCUSSION THREAD

**Discussion by Bocko001** [Mr-MIBonk/M.I.B._More-Incredible-Bash#503] [General]
Hello
is it safe to use patch, because in folder there is experimental.txt which hide inside ''This is strictly experimental, get in contact with M.I.B team before using this patch!''?
![byg](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/174388439/952fb5ff-da48-414e-96a4-1a6f144b7349)

**Comment by chukhapps-cloud** [comment]
It is working or you not trying?

**Comment by zbadguy** [comment]
Hello. Did anyone try this patch? Is it safe?
